build.gradle.kts

```diff
@@ -40,17 +40,25 @@ measureBuilds {
 
 spotless {
     kotlin {
-        target("**/*.kt")
-        targetExclude("$buildDir/**/*.kt")
-        targetExclude("bin/**/*.kt")
+        target(
+            "app/src/**/*.kt",
+            "automotive/src/**/*.kt",
+            "modules/src/**/*.kt",
+            "wear/src/**/*.kt",
+            "build.gradle.kts",
+            "app/build.gradle.kts",
+            "automotive/build.gradle.kts",
+            "modules/**/build.gradle.kts",
+            "wear/build.gradle.kts",
+        )
```

Comment:
I just realised I added an unused import. Adding the `build.gradle.kts` files to Spotless helped.

<img width="543" alt="Screenshot 2024-05-21 at 5 21 51 PM" src="https://github.com/Automattic/pocket-casts-android/assets/308331/804877ae-6cc6-4581-b1d0-bcee3f62a0df">